internal/kokoro/synth/synth.sh

```diff
@@ -7,6 +7,13 @@
 set -e -x
 
 cd $(dirname $0)/../../..
+
+# Install Go 1.15
```

Comment:
Seems like this should be updating some container image, rather than reinstalling Go?

Reply:
This is not run in a container today.